src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -349,17 +404,26 @@ func (s *Service) sendToRooms(cli *gomatrix.Client, feedURL string, feed *gofeed
 }
 
 func itemToHTML(feed *gofeed.Feed, item gofeed.Item) gomatrix.HTMLMessage {
+	body := fmt.Sprintf("<strong>%s</strong>:<br><a href=\"%s\"><strong>%s</strong></a>",
+		html.EscapeString(feed.Title), html.EscapeString(item.Link), html.EscapeString(item.Title))
+	if item.Author != nil {
+		if len(item.Author.Email) > 0 {
+			body += fmt.Sprintf(" by <a href=\"mailto:%s\">%s</a>", html.EscapeString(item.Author.Email),
+				html.EscapeString(item.Author.Name))
```

Comment:
If the author has an email but no name this will be a problem. Should probably default to the email in that case.

Reply:
Done!